src/controllers/aer_controller.hpp

```diff
@@ -963,7 +963,11 @@ Result Controller::execute(std::vector<Circuit> &circuits,
       parallel_nested_ = true;
 
       // nested should be set to zero if num_threads clause will be used
+#if defined(_WIN64) || defined(_WIN32)
       omp_set_nested(0);
+#else
+      omp_set_max_active_levels(0);
+#endif
```

Comment:
This maybe wants to gate on `#if defined(_OPENMP) && _OPENMP >= 200805` instead (that's a check that we've got OpenMP 3.0+) - it's a bit more precise about what we actually need.